app/src/main/java/co/nimblehq/compose/crypto/ui/screens/detail/DetailScreen.kt

```diff
@@ -87,10 +116,8 @@ fun DetailScreen() {
                         top.linkTo(currentPrice.bottom)
                         linkTo(start = parent.start, end = parent.end)
                     },
-                // TODO: Remove dummy image when work on Integrate.
-                priceChangePercent = "6.21",
-                displayForDetailPage = true,
-                isPositiveNumber = true
+                priceChangePercent = coinDetailUiModel.priceChangePercentage24hInCurrency,
```

Comment:
we should format this to 2 decimals only

![image](https://user-images.githubusercontent.com/16315358/193008445-209bceb1-c9f7-4614-977a-c0ed74854fb3.png)